store/store.go

```diff
@@ -91,6 +91,11 @@ func (s *BasicStore) WaitToDoJob() (chan *gotell.Job, error) {
 	return receiver, nil
 }
 
+func (s *BasicStore) StopWaiting(receiver chan *gotell.Job) {
```

Comment:
I take it this was the bug you were talking about?

Reply:
Yes, have a look at TestClosingWorkerDoesntBreakThings